src/pages/colinks/NotFound.tsx

```diff
@@ -0,0 +1,10 @@
+import { Flex, Text } from 'ui';
+
+export const NotFound = () => {
```

Comment:
@wingfeatherwave Merging this now - but wanna and an image to this? I was thikning something like looking under a rock with this text but do whatever you want